std/math/bits/conversion_ternary.go

```diff
@@ -56,7 +55,7 @@ func fromTernary(api frontend.API, digits []frontend.Variable, opts ...BaseConve
 
 func toTernary(api frontend.API, v frontend.Variable, opts ...BaseConversionOption) []frontend.Variable {
 	// parse options
-	nbBits := api.Compiler().Curve().Info().Fr.Bits
+	nbBits := api.Compiler().Field().BitLen()
```

Comment:
If implement `BitLen()` in interface (described in other comment), then can replace.